registry/zookeeper/registry.go

```diff
@@ -150,7 +150,11 @@ func (r *zkRegistry) InitListeners() {
 
 // CreatePath creates the path in the registry center of zookeeper
 func (r *zkRegistry) CreatePath(path string) error {
-	return r.ZkClient().Create(path)
+	err := r.ZkClient().Create(path)
+	if err != nil && err != zk.ErrNodeExists {
```

Comment:
IMO, using errors.Is() is better than `!=`.

Reply:
I think != is just ok